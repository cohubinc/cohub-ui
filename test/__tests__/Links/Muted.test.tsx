import React from "react";
import { shallow } from "enzyme";
import { Link } from "dist";

describe("Muted", () => {
  it("renders without crashing", async () => {
    const wrapper = shallow(<Link.Muted to="/custom/path" />);
    expect(wrapper).toMatchSnapshot();
  });
});
