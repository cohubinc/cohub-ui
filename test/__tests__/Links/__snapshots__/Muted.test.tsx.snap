// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Muted renders without crashing 1`] = `
<Base
  styled={false}
  to="/custom/path"
>
  <Typography
    muted={true}
  />
</Base>
`;
